helix-tui/src/buffer.rs

```diff
@@ -583,6 +582,35 @@ impl Buffer {
         }
         updates
     }
+
+    /// Apply shade to all cells in area.
+    ///
+    /// shade = 0 -> set text modifier DIM
+    /// shade < 0 -> darken rgb color
+    /// shade > 0 -> lighten rgb color
```

Comment:
Regular line breaks in doc comments might not render as line breaks (CommonMark [§6.8](https://spec.commonmark.org/0.30/#hard-line-breaks))

```suggestion
    /// - `shade = 0`: set text modifier `DIM`
    /// - `shade < 0`: darken rgb color
    /// - `shade > 0`: lighten rgb color
```